src/service/AggregateTransactionService.ts

```diff
@@ -74,6 +82,41 @@ export class AggregateTransactionService {
             );
     }
 
+    /**
+     * Get total multisig account cosigner count
+     * @param address multisig account address
+     * @returns {Observable<number>}
+     */
+    public getMaxCosignatures(address: Address): Observable<number> {
+        return this.multisigRepository.getMultisigAccountGraphInfo(address).pipe(
+            map((graph) => {
+                let count = 0;
+                graph.multisigAccounts.forEach((multisigInfo) => {
+                    multisigInfo.map((multisig) => {
+                        count += multisig.cosignatories.length;
+                    });
+                });
+                return count;
+            }),
+        );
+    }
+
+    /**
+     * Get next cosignature allowed per aggregate
+     * @returns {Observable<number>}
+     */
+    public getNetworkMaxCosignaturesPerAggregate(): Observable<number> {
+        return this.networkRepository.getNetworkProperties().pipe(
+            map((properties) => {
+                console.log(!properties.plugins.aggregate?.maxCosignaturesPerAggregate);
+                if (!properties.plugins.aggregate?.maxCosignaturesPerAggregate) {
+                    throw new Error('Cannot get maxCosignaturesPerAggregate from network properties.');
+                }
+                return parseInt(properties.plugins.aggregate?.maxCosignaturesPerAggregate);
```

Comment:
A number may be `1'000`, remove possible `'`

http://api-01.us-west-1.symboldev.network:3000/network/properties

Reply:
will that ever be over 1000? @dgarcia360